src/test/suite/extension.test.ts

```diff
@@ -1,15 +1,62 @@
 import * as assert from 'assert';
+import * as proxyquire from 'proxyquire';
+import { MockingUtils, MockingExec } from './MockingUtils';
 
 // You can import and use all API from the 'vscode' module
 // as well as import your extension to test it
-import * as vscode from 'vscode';
-// import * as myExtension from '../../extension';
+//import * as vscode from 'vscode';
+const vscode = require('vscode');
```

Comment:
Why use a require instead of an import?

Reply:
To remember we can use proxyquire there down the road.